### PR TITLE
[OSDOCS-3061] ShiftStack control plane migration

### PR DESCRIPTION
**Pull request by maxwelldb**
Context: https://issues.redhat.com/browse/OSDOCS-3061

Preview: https://deploy-preview-41367--osdocs.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-working.html#nodes-control-plane-osp-migrating_nodes-nodes-working

### CODE REVIEW

**Comment by maxwelldb** [comment]
FYI @mandre

File: modules/nodes-control-plane-osp-migrating.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+set -x
+
+declare -r node_name="$1"
+declare server_id
+server_id="$(openstack server list --all-projects -f value -c ID -c Name | grep "$node_name" | cut -d' ' -f1)"
+readonly server_id
+
+# Drain the node
+oc adm cordon "$node_name"
+oc adm drain "$node_name" --delete-emptydir-data --ignore-daemonsets
```
Comment (mandre) [inline modules/nodes-control-plane-osp-migrating.adoc:46]: We might have to use the `--force` option in the drain command due to bug https://bugzilla.redhat.com/show_bug.cgi?id=2047257. I'll let @pierreprinetti sort it out next week when he's back (and I'm off).

**Review by mandre — COMMENTED** [review]
Looks good. I just left a note about a potential change to the script, we might have to change it a little bit. Pierre will be able to confirm that next week.

File: modules/nodes-control-plane-osp-migrating.adoc (hunk 1 above)
Comment (pierreprinetti) [reply modules/nodes-control-plane-osp-migrating.adoc:46]: cf https://github.com/openshift/installer/pull/5618

Comment (maxwelldb) [reply modules/nodes-control-plane-osp-migrating.adoc:46]:
```suggestion
oc adm drain "$node_name" --delete-emptydir-data --ignore-daemonsets --force
```

All good? @mandre @pierreprinetti

File: modules/nodes-control-plane-osp-migrating.adoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * TBD
```
Comment (maxwelldb) [inline modules/nodes-control-plane-osp-migrating.adoc:3]:
```suggestion
// * nodes/nodes/nodes-nodes-working.adoc
```

**Comment by eurijon** [comment]
lgtm

File: modules/nodes-control-plane-osp-migrating.adoc (hunk 1 above)
Comment (mandre) [reply modules/nodes-control-plane-osp-migrating.adoc:46]: Yes please add the `--force` flag, it's not completely clear to me at the moment why we need it -- it looks like the `guard` pods are a late addition in 4.10 and will not drain properly unless passing the `--force` flag.

We'll investigate in 4.11 and possibly remove this flag in the future once we understand the issue better.

**Comment by maxwelldb** [comment]
@eurijon Any issues with the `--force` flag added by https://github.com/openshift/openshift-docs/pull/41367/commits/899e4e6672c8c5d1cb2ea0c6947f7440be375fce?

**Comment by eurijon** [comment]
> @eurijon Any issues with the `--force` flag added by [899e4e6](https://github.com/openshift/openshift-docs/commit/899e4e6672c8c5d1cb2ea0c6947f7440be375fce)?

Hey @maxwelldb , no, it's working fine now :)

File: modules/nodes-control-plane-osp-migrating.adoc [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+readonly server_id
+
+# Drain the node
+oc adm cordon "$node_name"
+oc adm drain "$node_name" --delete-emptydir-data --ignore-daemonsets --force
+
+# Power off the server
+oc debug "node/${node_name}" -- chroot /host shutdown -h 1
+
+# Verify the server is shutoff
```
Comment (maxwelldb) [inline modules/nodes-control-plane-osp-migrating.adoc:51]:
```suggestion
# Verify the server is shut off
```

**Review by lpettyjo — COMMENTED** [review]
I notice that there are 3 commits. Please squash to one.

Purhaps I misunderstand what these prerequisites are, but these don't seem like prerequisites, but rather explanation of terms. Maybe re-label?

Otherwise, LGTM.

**Pull request merged**